packages/framework/src/Migrations/Version20180603135342.php

```diff
@@ -14,7 +14,7 @@ public function up(Schema $schema)
     {
         $currenciesCount = $this->sql('SELECT count(*) FROM currencies')->fetchColumn(0);
         if ($currenciesCount <= 0) {
-            $this->sql('INSERT INTO currencies (id, name, code, exchange_rate) VALUES (1, \'Česká koruna\', \'CZK\', 1)');
+            $this->sql('INSERT INTO currencies (id, name, code, exchange_rate) VALUES (1, \'Euro\', \'EUR\', 1)');
```

Comment:
Changing migrations isn't really safe, a new migration should be generated if necessary.

Reply:
I know, but i this case... New migration would cause really big problem for existing projects. And new projects install this migration with right currency.

Reply:
But this introduces a major BC break. If this PR would be just about changing data fixture, it would modify only the `project-base` repository and thus would not complicate upgrading.

Reply:
This changes (whole PR) will be complicated for projects... It's my point of view... And I want to do final solution, not half-solution because BC break. Will it be updated in next major releases without confusing code?

Reply:
In my opinion, it will be similarly difficult both ways. When the modifications are only in the project repository, users will at least have the option to omit this on their project if they choose to (as the project will still work without the changes).

To be honest, I don't really see having CZK with ID=1 and EUR with ID=2 as a half-solution. This change should be about domains and that can be resolved using demo-data only.